docs/index.html

```diff
@@ -203,10 +204,10 @@ <h2 class="to-animate">Build the trusted economy in IoT, securing data with IOTA
 			<div class="container">
 				<div class="row">
 					<div class="col-md-12 section-heading text-center">
-						<h2 class="to-animate"><span>We provide</span></h2>
+						<h2 class="to-animate"><span>Our Service</span></h2>
 						<div class="row">
 							<div class="col-md-8 col-md-offset-2 subtext">
-								<h3 class="to-animate">Facing to the upcoming Machine Economy, over 5 Trilion transactions and microtrasactions happens everday by 2020, BiiLabs works with <a href="https://www.iota.org/">IOTA Foundation</a> closely for next revolution (Internet of Trusted) in each industry. </h3>
+								<h3 class="to-animate">Facing with the ongoing “Machine Economics,” over 5 Trillion transactions and microtransactions taking place daily by 2020, BiiLabs intends to provide distributed technology as next-generation leverage for each industry, in cooperation with IOTA Foundation. </h3>
```

Comment:
Why "transactions and microtransactions" here?

Reply:
Change "distributed technology" to "distributed ledger infrastructure".

Reply:
according to previous version, I suppose this usage means to emphasize on the realization of micro-payment records